lib/services/sku-pack-service.js

```diff
@@ -444,28 +462,30 @@ function skuPackServiceFactory(
      * @param  {String}     fromRoot The path to the configuration file
      * @return {Promise}
      */
-    SkuPackService.prototype.validatePack = function(contents, fromRoot) {
-        try {
+    SkuPackService.prototype.validatePack = function(contents, fromRoot, options) {
+        options = options || {};
+
+        return Promise.try(function() {
             var conf = JSON.parse(contents);
```

Comment:
Could you please add a try/catch for JSON.parse? so the invalid JSON format error can be explicitly threw out to user,  I had said this was an enhance opportunity in this issue https://github.com/RackHD/RackHD/issues/190